frontend/src/components/dashboard/create/sms/SMSCredentials.tsx

```diff
@@ -130,9 +159,13 @@ const SMSCredentials = ({
         </p>
         <SMSValidationInput
           onClick={handleValidateCredentials}
```

Comment:
I think you never passed the label to the handler? https://github.com/opengovsg/postmangovsg/blob/823a4e57f8852d83fc4c98527770922fb4c83721/frontend/src/components/dashboard/create/sms/SMSCredentials.tsx#L81-L86